streaming/src/data_writer.cc

```diff
@@ -447,5 +431,54 @@ void DataWriter::EmptyMessageTimerCallback() {
   }
 }
 
+void DataWriter::NotifyConsumedByRefresh(ProducerChannelInfo &channel_info) {
```

Comment:
Is `NotifyConsumedByRefresh` means `refresh` and then  `NotifyConsumedItem`?
Maybe better to use `RefreshAndNotifyConsumed`?